docs/content/getting-started/setup-openfga/kubernetes-setup.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Kubernetes Setup Guide
+description: Setting up an OpenFGA server with Kubernetes
+sidebar_position: 2
+slug: /getting-started/setup-openfga/kubernetes
+---
+
+import {
+  DocumentationNotice,
+  RelatedSection,
+} from "@components/Docs";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# ☸️ Setup OpenFGA with Kubernetes
+To deploy OpenFGA into a Kubernetes environment you can use the official [OpenFGA Helm chart](https://artifacthub.io/packages/helm/openfga/openfga). Please refer to the official documentation on Artifact Hub for the Helm chart for more instructions.
```

Comment:
Do we need to add in more details (i.e., for example, how to configure certain parameters/whether to open port for playground)?  Otherwise, this page feels a bit empty.

Reply:
The purpose of that is to live in the Artifact Hub package documentation for the Helm chart instead of duplicating that content across sources. If we feel the documentation is lacking for those details, then let's update that in the OpenFGA Helm chart documentation.